README.md

```diff
@@ -51,18 +51,39 @@ Use `inv -l` to get a list of all the available invoke commands.
 
 More information on how to work with Docker and how to manage Python dependencies are available in the [local development](docs/local_development.md) part of the documentation.
 
+## Code style
+
+To ensure a consistent code style and quality, we use linters and formatters.
+
+### Linting
+
+To check the code base for quality and style issues run `inv lint`.
+This will run all configured linters.
+You can run the linters individually with, e.g. `inv lint-js` for JavaScript only.
+Check available commands with `inv -l`.
+
+### Formatting
+
+If `inv lint` shows linting errors for either JS/JSX or CSS/SCSS, you can run `inv format` for fix style issues.
+`inv format` should automatically fix most formatting issues in JS and CSS files.
+
 ## Testing
 
-### Code tests
+### Unit tests
+
+When relevant, we encourage you to write tests.
 
-When relevant, we encourage you to write tests. You can run the tests using `inv test`, or you can run the Node and Python testing suites separately:
+You can run the tests using `inv test`.
+This will the full test suite.
+To run only a subset or a specific Python test, you can use following command:
 
-- Run Node tests: `inv test-node`
-- Run Python tests: `inv test-python`
+```console
+inv manage "test <dotted-path-to-your-test>"
```

Comment:
Thanks for adding this Tibor, I think this is a really useful thing to have in the docs